transform/README.md

```diff
@@ -68,6 +69,13 @@ We are using .env (in the root of the EOSC Transform Service) to store user-spec
   - `S3_ENDPOINT`: `str = ""` - S3 endpoint. Example: `https://s3.cloud.com`.
   - `S3_BUCKET`: `str = ""` - S3 bucket. Example: `ess-mock-dumps`.
 
+##### Relations
+- `CREATE_RELATIONS`: `bool = False` - A flag which indicates if relations should be processed.
```

Comment:
I added this flag because it allows us to test new fields added into Solr without having to reprocess relations. Without the flag, we would have to wait an additional ~6 minutes each time. Assuming that relations were processed before.